src/api/app/components/copy_to_clipboard_input_component.html.haml

```diff
@@ -1,5 +1,5 @@
 -# TODO: Relying on a form builder is a bit weird. We could perhaps directly use `text_field_tag` instead and pass an input value to the component.
-= @form_builder.text_field(:string_readonly, id: 'copy-to-clipboard-readonly', class: 'form-control', readonly: true)
+%input.form-control{value: @token_string, disabled: true}
```

Comment:
I suggest using the [text_field_tag](https://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-text_field_tag) helper instead. The id `copy-to-clipboard-readonly` should still be used for the JavaScript to keep working.